cypress/fixtures/analysis.json

```diff
@@ -59,58 +52,182 @@
             {
                 "name": "File system - Java IO",
                 "category": "mandatory",
-                "source": "None",
+                "source": ["None"],
```

Comment:
I saw that you renamed the file `source` to `sources`, but it has to be renamed in the json file as well

Reply:
Fixed, thanks for noticing this